### PR TITLE
fix(WelcomeStepsModal): Postal code entry not working when other moda…

### LINKED ISSUE #348: Intro - Postal code input field is blocked if map data cannot be loaded
**Describe the bug**
Opening the app on a fresh browser (clear history) when the data for the map cannot be loaded "Keine Daten für den ausgewählten Zeitraum." leads to the postal code field on the last intro screen to be blocked (nothing can be entered).

### PR DESCRIPTION
**Pull request by kantimam**
fixed #348 and overflow issues inside the WelcomeStepsModal

### CODE REVIEW

File: src/components/CovMap.tsx [hunk 1]
```diff
@@ -294,12 +294,13 @@ export const CovMap = () => {
       <Legend />
       <Dialog
         aria-labelledby="simple-dialog-title"
-        open={!datasetFound}
+        open={!datasetFound || true}
```
Comment (DanielHabenicht) [inline src/components/CovMap.tsx:297]:
```suggestion
        open={true}
```

Comment (DanielHabenicht) [reply src/components/CovMap.tsx:297]: no need to make it more complex than it is 👍

File: src/components/CovMap.tsx [hunk 2]
```diff
@@ -294,12 +294,13 @@ export const CovMap = () => {
...
         style={{
           zIndex: 1190,
           touchAction: "none",
           position: "absolute",
         }}
+        disableEnforceFocus
```
Comment (DanielHabenicht) [inline src/components/CovMap.tsx:303]: Does this also mean that the keyboard won't open automatically on mobile?

File: src/components/CovMap.tsx (hunk 1 above)
Comment (kantimam) [reply src/components/CovMap.tsx:297]: ohh wow guess he didn't save... should be only !datasetFound....   OR true was only for debugging to always show it

File: src/components/CovMap.tsx (hunk 2 above)
Comment (kantimam) [reply src/components/CovMap.tsx:303]: na it means if the second modal opens on top of the welcomeStepsModal it won't hijack the the focus from the welcomeStepsModal and it's input field.

File: src/components/CovMap.tsx (hunk 1 above)
Comment (kantimam) [reply src/components/CovMap.tsx]: should be fine now :)

**Review by DanielHabenicht — APPROVED** [review]

**Pull request merged**